### ISSUE TITLE
how to specify the weight map for loss function in config?

### ISSUE THREAD

**Issue by YilinLiu97** [NifTK/NiftyNet#84]
I want to specify the weight map for loss function (e.g. cross-entropy) to handle the imbalanced class problem. How can I specify that in the config? 

Thanks!

**Comment by YilinLiu97** [comment]
I've checked out the segmentation_application.py and it looks like that there is no way to pass weight values...(?) Then how can users handle the imbalanced class problem?

**Comment by danieltudosiu** [comment]
You need to create another input and name for example Weights and then pass that input (which can be found in the data_dict parameter of segmentation application to the loss function as a variable for the parameter weight_map.
